hw2/04-background.js

```diff
@@ -1 +1,39 @@
-// Add your code here
+const toggleButton = document.getElementById("toggleButton");
+const input = document.getElementById("number");
+const container = document.getElementById("container");
+
+getRandomNumber = (max) => {
+	return Math.floor(Math.random() * max);
+};
+
+changeBackgroundColor = (maxRGB = 255) => {
```

Comment:
I think you'll need to increase your maxRGB to 256 to get all numbers 0-255 for your Math.random() function.

Reply:
Done